### PR TITLE
docs: update sitemap and meta

### PR DESCRIPTION
**Pull request by hsuanxyz**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/NG-ZORRO/ng-zorro-antd/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] Application (the showcase website) / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

File: components/ng-package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "../node_modules/ng-packagr/ng-package.schema.json",
-  "dest": "../publish",
+  "dest": "../publish_ivy",
```
Comment (hsuanxyz) [inline components/ng-package.json]: 在构建文档是无法使用 non-ivy 包，在发布时不建议使用 ivy 包，所以现在 `publish_ivy` 用于构建文档，`publish` 用于发布 😂

File: scripts/site/_site/doc/index.html [hunk 2]
```diff
@@ -4,7 +4,30 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <meta name="apple-mobile-web-app-capable" content="yes">
+    <meta name="apple-mobile-web-app-status-bar-style" content="translucent">
+    <meta name="mobile-web-app-capable" content="yes">
+    <meta name="author" content="NG-ZORRO Team">
     <meta name="description" content="An enterprise-class UI design language and Angular-based implementation with a set of high-quality Angular components, one of best Angular UI library for enterprises">
+    <!--Twitter card metadata-->
+    <meta name="twitter:card" content="summary">
```
Comment (wendzhue) [inline scripts/site/_site/doc/index.html]: Nice!

File: components/ng-package.json (hunk 1 above)
Comment (wendzhue) [reply components/ng-package.json]: 😂

File: setup-shared-libs.sh [hunk 3]
```diff
@@ -2,11 +2,11 @@ TARGET="$1"
...
 
 mkdir -p "$TARGET/ng-zorro-antd"
-rsync -av --exclude=".*" "$ROOT/publish/" "$TARGET/ng-zorro-antd" > /dev/null 2>&1
+rsync -av --exclude=".*" "$ROOT/publish_ivy/" "$TARGET/ng-zorro-antd" > /dev/null 2>&1
 
 for lib in "@angular/animations" "@angular/cdk" "@angular/common" "@angular/compiler" "@angular/core" "@angular/forms" "@angular/platform-browser" "@angular/platform-browser-dynamic" "@angular/router" "@angular/cli" "@angular/compiler-cli" "rxjs" "typescript" "zone.js" "date-fns" "core-js"
 do
   mkdir -p "$TARGET/$lib"
-  rsync -av --exclude=".*" "$ROOT/node_modules/$lib/" "$TARGET/$lib" > /dev/null 2>&1
+  rsync -av --exclude=".*" "$ROOT/node_modules/$lib/" 我"$TARGET/$lib" > /dev/null 2>&1
```
Comment (vthinkxie) [inline setup-shared-libs.sh]: 😂

File: package.json [hunk 4]
```diff
@@ -103,6 +103,7 @@
     "sitemap": "^4.1.1",
     "ts-node": "~8.3.0",
     "tslint": "~5.18.0",
+    "tsickle": "^0.37.1",
```
Comment (vthinkxie) [inline package.json]:
```suggestion
```

File: scripts/site/_site/doc/index.html [hunk 5]
```diff
@@ -4,7 +4,30 @@
...
+    <meta name="twitter:title" content="Angular Material">
+    <meta name="twitter:description" content="An enterprise-class UI design language and Angular-based implementation with a set of high-quality Angular components, one of best Angular UI library for enterprises">
+    <meta name="twitter:image" content="https://ng.ant.design/assets/img/site-preview.png">
+    <!--Open Graph-->
+    <meta property="og:url" content="https://ng.ant.design/">
+    <meta property="og:type" content="website">
+    <meta property="og:title" content="NG-ZORRO - Ant Design Of Angular">
+    <meta property="og:site_name" content="NG-ZORRO - Ant Design Of Angular">
+    <meta property="og:locale" content="en_US">
+    <meta property="og:description" content="An enterprise-class UI design language and Angular-based implementation with a set of high-quality Angular components, one of best Angular UI library for enterprises">
```
Comment (vthinkxie) [inline scripts/site/_site/doc/index.html]: should update according to lang
https://angular.io/api/platform-browser/Meta

File: scripts/site/_site/doc/index.html [hunk 6]
```diff
@@ -12,7 +12,7 @@
     <meta name="twitter:card" content="summary">
     <meta name="twitter:site" content="@NG_ZORRO">
```
Comment (vthinkxie) [inline scripts/site/_site/doc/index.html]: @NG_ZORRO->@ng_zorro

**Pull request merged**